app/src/main/java/com/example/footballmanager/screens/result_screens/success_components/MatchScoreTab.kt

```diff
@@ -81,54 +81,57 @@ fun MatchScoreTab(
         ) {
             var previousLeagueName = defaultValue
             for (i in 0..lastLoadedIndex) {
-                val item = fixture.responseBody[i]
-                val leagueName = item.league?.name ?: defaultValue
-                val leagueLogo = item.league?.logo ?: defaultValue
-                val nameTeamHome = item.teams?.home?.name ?: defaultValue
-                val nameTeamAway = item.teams?.away?.name ?: defaultValue
-                val scoreTeamHome = item.goals?.home?.toString() ?: defaultValue
-                val scoreTeamAway = item.goals?.away?.toString() ?: defaultValue
-                val logoTeamHome = item.teams?.home?.logo ?: defaultValue
-                val logoTeamAway = item.teams?.away?.logo ?: defaultValue
-                val short = item.fixture?.status?.short ?: defaultValue
-                var date = item.fixture?.date ?: defaultValue
+                val item = fixtureDataWrapper.responseBody[i]
+                with(item) {
+                    val leagueName = league?.name ?: defaultValue
+                    val leagueLogo = league?.logo ?: defaultValue
+                    val nameTeamHome = teams?.home?.name ?: defaultValue
+                    val nameTeamAway = teams?.away?.name ?: defaultValue
+                    val scoreTeamHome = goals?.home?.toString() ?: defaultValue
+                    val scoreTeamAway = goals?.away?.toString() ?: defaultValue
+                    val logoTeamHome = teams?.home?.logo ?: defaultValue
+                    val logoTeamAway = teams?.away?.logo ?: defaultValue
+                    val short = fixture?.status?.short ?: defaultValue
+                    var date = fixture?.date ?: defaultValue
 
-                //date formating
-                if (date != defaultValue) {
-                    try {
-                        val offsetDateTime =
-                            OffsetDateTime.parse(date, DateTimeFormatter.ISO_OFFSET_DATE_TIME)
 
-                        val dateDate = offsetDateTime.toLocalDate()
-                            .format(DateTimeFormatter.ofPattern("dd/MM"))
-                        date = dateDate.toString()
-                    } catch (e: Exception) {
-                        Toast.makeText(ctx, e.toString(), Toast.LENGTH_SHORT).show()
+                    //date formating
+                    if (date != defaultValue) {
+                        try {
```

Comment:
You could use [`runCatching`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/run-catching.html) here